absl/random/internal/randen_engine.h

```diff
@@ -105,7 +105,7 @@ class alignas(8) randen_engine {
     // Zeroes the inner state and fills the outer state with seed_value to
     // mimic the behaviour of reseed
     auto* begin = state();
-    std::fill(begin, begin + kCapacityT, 0);
+    std::fill_n(begin, kCapacityT, 0);
```

Comment:
This one is less readable in my opinion because the next line has a `std::fill` on it that starts in the middle of the array. It's easier to read when they consistently do the same thing in the same way. I think these should either both be `std::fill` or `std::fill_n`, and I'd lean towards `std::fill`.

Reply:
Fixed!